src/app/clusters/temperature-measurement-server/temperature-measurement-server.c

```diff
@@ -0,0 +1,64 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
```

Comment:
Nit: since the apache license bit is identical, should we just merge the 2 copyright lines and save some wall of comment text?

Reply:
I'm fine with doing that. But I would rather do it in a separated PR. There is a bunch of files that use this licence format and I may have to double check with them.